spec/models/transaction_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Transaction, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of(:amount) }
+    it { should validate_numericality_of(:amount) }
```

Comment:
We should be consistent and use `is_expected` instead of `should` here. Also, looks like this is missing the spec for the `greater_than` part.